packages/mobile/src/app/App.tsx

```diff
@@ -66,6 +67,23 @@ export class App extends React.Component<Props> {
   async componentDidMount() {
     await ValoraAnalytics.init()
 
+    // Handles opening Clevertap deeplinks when app is closed / in background
+    CleverTap.getInitialUrl(async (err: any, url: any) => {
+      if (err) {
+        Logger.debug('App', err)
+      } else if (url) {
+        await this.handleOpenURL({ url }, true)
+      }
+    })
+
+    // Handles opening Clevertap deeplinks when app is open
+    CleverTap.addListener('CleverTapPushNotificationClicked', async (event: any) => {
+      if (event.customExtras['wzrk_dl']) {
+        const url = event.customExtras['wzrk_dl']
+        await this.handleCleverTapUrl(url, true)
```

Comment:
Can't we just do the same as in `getInitialUrl` and call `await this.handleOpenURL({ url }, true)` instead of creating a new function?

Reply:
I can give it another try like that. Unfortunately it didn't appear that the CleverTap event had a url field to be deconstructed.

Reply:
Yup, but this is the opposite of deconstruction, this is building an object with a single `url` field, just like you did a few lines above, unless I'm reading this wrong 🤔

Reply:
Fixing in https://github.com/valora-inc/wallet/pull/2012/commits/c5ad74ee98e6a3fa0623afd73fad0d892ed3700c.